### PR TITLE
✨ [PROD-1521] Add Carts API docs

### PR DESCRIPTION
**Pull request by knor-el-snor**
https://github.com/ShopBonsai/bonsai/pull/15312 is related, but does not necessarily have to get deployed first as the public API endpoints won't work yet anyway

Did not add the menu as we don't have public API working yet, but the pages can be accessed individually

### CODE REVIEW

**Comment by knor-el-snor** [comment]
Copied over the auto-generated spec file from Carts API, the rest is auto-generated using `re-gen`

**Comment by knor-el-snor** [comment]
Seems like something's off when running `yarn build` but can't immediately see what. Will have to investigate further

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const config = {
   title: 'Bonsai Documentation',
   tagline: '',
-  url: 'https://your-docusaurus-test-site.com',
+  url: 'https://docs.shopbonsai.ca',
```
Comment (knor-el-snor) [inline docusaurus.config.js:11]: Updated the URL in our config while I was at it

File: yarn.lock [hunk 2]
```diff
@@ -2953,9 +2953,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001400, caniuse-lite@^1.0.30001426:
```
Comment (knor-el-snor) [inline yarn.lock:2955]: Updated caniuse

**Review by yashafromrussia — APPROVED** [review]

**Pull request merged**